src/main/webapp/app/quiz-statistics/quiz-statistic.module.ts

```diff
@@ -35,8 +37,10 @@ const ENTITY_STATES = [...quizStatisticRoute];
         MultipleChoiceQuestionStatisticComponent,
         DragAndDropQuestionStatisticComponent,
         ShortAnswerQuestionStatisticComponent,
+        QuizStatisticsFooterComponent,
```

Comment:
I see that you follow the conventions of the module, but I think `QuizStatisticsFooterComponent` is not an `entryComponent` (https://angular.io/guide/entry-components). EntryComponents are only necessary if the component is created outside of the template.